lib/rack/cache/redis_entitystore.rb

```diff
@@ -8,21 +7,26 @@ class RedisBase < self
         # The underlying ::Redis instance used to communicate with the Redis daemon.
         attr_reader :cache
 
+        class << self
```

Comment:
Not sure if this is the best way to configure default_ttl. I would much rather continue to use `Rack::Cache::DEFAULT_TTL` if a TTL is not provided in the initial configuration, then set `@default_ttl` on the actual instance each time based on whether an override was given.